src/main/java/aeneas/models/PuzzleLevel.java

```diff
@@ -13,8 +19,8 @@
 
   PuzzleBoard board;
 
-  private int moves;
-
+  private int movesAllowed;
```

Comment:
We need to initialize movesAllowed in the model (it's limited to > 1 in the view's spinner, but if you don't touch the spinner then the moves in the model is still 0).